install_win.ps1

```diff
@@ -0,0 +1,32 @@
+If( !(Test-Path "C:\Python27\Scripts\pip.exe") ){
+  Write-Host "[+] Downloading and installing python 2.7.13 (with pip)"
+  (New-Object System.Net.WebClient).DownloadFile('https://www.python.org/ftp/python/2.7.13/python-2.7.13.amd64.msi','deps\python-2.7.13.amd64.msi')
+  $process="deps\python-2.7.13.amd64.msi"
+  $args="/qb"
+  Start-Process $process -ArgumentList $args -Wait
+}
+
+Write-Host "[+] Downloading and installing VCForPython 2.7"
```

Comment:
The check is ensuring pip is installed since it is used to install dependencies that don't properly install using the default python setup.py install